lib/led-matrix/src/LEDMatrix.cpp

```diff
@@ -0,0 +1,74 @@
+#include "LEDMatrix.h"
+
+#include "mbed.h"
+
+LEDMatrix::LEDMatrix(PinName R, PinName G, PinName B)
+    : m_RChannel(R),
+      m_GChannel(G),
+      m_BChannel(B),
+      m_led_state(HWBRIDGE::LEDMATRIX::LEDMatrixState::OFF),
+      m_continue_flashing(false) {
+  m_lightsThread.start(callback(this, &LEDMatrix::flashing));
+}
+
+LEDMatrix::~LEDMatrix() {
+  setSolidColor(0, 0, 0);
+}
+
+void LEDMatrix::setState(HWBRIDGE::LEDMATRIX::LEDMatrixState state) {
+  m_led_state = state;
```

Comment:
This `m_led_state` member var isn't needed atm. We might use it in the future though, but if we don't end up using it make sure to remove it later

Reply:
Right, I just removed it for now